pages/home/auth/secure-auth-production.mdx

```diff
@@ -0,0 +1,127 @@
+---
+title: "Secure Auth in Production"
+description: "How to secure and brand your auth flows in production"
+---
+
+# Secure and Brand the Auth Flow in Production
+
+To keep your users safe, Arcade.dev performs a session verification check when a tool is authorized for the first time. This check verifies that the user who is authorizing the tool is the same user who started the authorization flow, which helps prevent phishing attacks.
+
+There are two ways to secure your auth flows with Arcade.dev:
+- Use the **Arcade session verifier** for development (enabled by default)
+- Implement a **custom session verifier** for production
+
+This setting is configured in the Arcade Dashboard.
+
+
+## Use the Arcade session verifier
+
+If you're building a proof-of-concept app or a solo project, use the Arcade session verifier. This option is on by default when you create a new Arcade.dev account.
+
+When you authorize a tool, you'll be prompted to sign in to your Arcade.dev account. If you are already signed in (to the Arcade Dashboard, for example), this verification will succeed silently.
+
+<Note>
+  Arcade's default OAuth apps can *only* be used with the Arcade session verifier.
+  If you are building a multi-user production app, you must obtain your own OAuth app credentials and add them to Arcade.
+  For example, [configure Google auth in the Arcade Dashboard](https://docs.arcade.dev/home/auth-providers/google#access-the-arcade-dashboard).
+</Note>
+
+The Arcade.dev session verifier helps keep your auth flows secure while you are building and testing your agent or app. When you're ready to share your work with others, implement a [custom session verifier](#set-up-a-custom-session-verifier) so your users don't need to sign in to Arcade.dev.
```

Comment:
Switch the order of these 2 paragraphs

Reply:
The `<Note>` para and this para?

Reply:
I'd like the explanation of the session verifier to come first